Add some contents to the README file (#166)
mia-platform/community: README.md

Head to the [Discussion Section][mia-community] to join!

</div>

# About this space

> The Mia-Platform community is an open space for everyone using our products, any question, information or feedback is welcome so feel free to join us!

Author: ```suggestion
The Mia-Platform community is an open space for everyone using our products; any question, information or feedback is welcome so feel free to join us!
```

# Mia-Platform Documentation

> If you to go solo you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs]

Reviewer: Manca forse questo?
```suggestion
If you wish to go solo you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs]
```

Author: I've applied a different suggestion but kept your fix, thanks

> If you to go solo you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs]

Author: ```suggestion
If you want to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs].
```

# Mia-Platform Marketplace

> The [Mia-Platform Markteplace][mia-marketplace] is the place where you can find everything you need to, from open source code templates to full-fledged microservices, you can customize the marketplace with your own components, if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!

Author: ```suggestion
The [Mia-Platform Marketplace][mia-marketplace] is the place where you can find everything you need to in order to speed up the development of your application! 

From open source code templates to full-fledged microservices, moreover you can customize the marketplace with your own components; if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!
```

Head to the [Discussion Section][mia-community] to join!

</div>

# About this space

> The Mia-Platform community is an open space for everyone using our products; any question, information or feedback is welcome so feel free to join us!

Reviewer 2: We should add a contributing file to link here, where we explain to not insert clients name or information and other reserved information (e.g. some secret values)

Author: That's a good insight, do you have time to write it?

I could bootstrap it but I guess the contributing rules can be a lot @akelity you might want to participate in it as well?

Author: I've added the file with a basic structure of paragraph I think may be useful; I did not write them all yet, feel free to add yours

# Mia-Platform Documentation

> If you wish to go solo and learn everything by yourself, you can find all the information you need inside [Mia-Platform Documentation Portal][mia-docs].

Author: ```suggestion
If you wish to go solo and learn everything by yourself, you can find all the information you need inside [Mia-Platform Documentation][mia-docs].
```

> If you wish to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs].

Reviewer 2: ```suggestion
If you wish to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation][mia-docs].
```

# Mia-Platform Marketplace

The [Mia-Platform Marketplace][mia-marketplace] is the place where you can find everything you need to in order to speed up the development of your application! 

> From open source code templates to full-fledged microservices, moreover you can customize the marketplace with your own components; if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!

Reviewer 3: ```suggestion
From open source code templates to full-fledged microservices, you can customize the Marketplace with your own components; if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!
```
